### ISSUE TITLE
Cannot generate completion when using split configs

### ISSUE THREAD

**Issue by surgiie** [DannyBen/bashly#177]
Hello again, 

Im not sure if im misconfiguring this on my end but it appears like completion generation is not working when using split configs:

```yaml
name: mycli
commands:
- import: src/configs/example.yml
```
The split config has a name defined

```yaml
name: example
short: e
help: Example command.
```

but when I tried to run `bashly add comp script` generates the following error is thrown:

```bash
Bashly::ConfigurationError
root.commands[0].name must be a string
```

I have confirmed that completion works as expected when the command config is defined in the `bashly.yml` file but it appears when using `import` the yaml data is not being recognized?

**Comment by DannyBen** [comment]
Thank you for the report. This is a small bug, for which I found the cause.
In fact - its not only completions, even if you just run `bashly validate` it will fail.

If you are able to test a version from GitHub, I will push a fix in a few minutes.

**Comment by surgiie** [comment]
Thanks for looking into this. I don’t have ruby installed on my system and I use bashly via docker. I can install it if needed if you need me to test this but I’m unable to at this moment.

**Comment by DannyBen** [comment]
Thanks, no need then. I will release a fix soon.

**Comment by DannyBen** [comment]
This is fixed and released in 0.7.5. Thanks for the report.
